syft/event/event.go

```diff
@@ -6,12 +6,10 @@ package event
 
 import (
 	"github.com/wagoodman/go-partybus"
-
-	"github.com/anchore/syft/internal"
 )
 
 const (
-	typePrefix    = internal.ApplicationName
+	typePrefix    = "syft"
```

Comment:
Events originating from this library, regardless of the actual application name make sense to be `"syft"`, I think

Reply:
though, this might motivate keeping the `internal.Syft` constant

Reply:
Note: I've removed the `internal.ApplicationName` and created the `internal.Syft` in the `cmd/internal` package, which isn't available here. There aren't any other locations where a hardcoded `"syft"` seems to be needed in the `syft/` packages from what I found.